client/js/_author/components/assessments/question_types/_question.jsx

```diff
@@ -55,9 +55,27 @@ export default class Question extends React.Component {
     this.props.updateItem({ id: item.id, ...newItemProperties });
   }
 
+  makeReflection(reflect) {
+    const { item } = this.props;
+    let type = 'multipleChoice';
+    if (reflect) {
+      type = item.multipleAnswer ? 'multipleReflection' : 'reflection';
+    }
+
+    this.props.updateItem({
+      id: item.id,
+      type,
+      question: {
+        type,
+        choices: true,
```

Comment:
No, but when we switch to reflection we need to update  values on the choice fields, which only happens if you pass in choices.